src/components/GradebookFilters/PercentGroup.jsx

```diff
@@ -22,7 +22,7 @@ const PercentGroup = ({
         {...{ value, disabled, onChange }}
       />
     </Form.Group>
-    <span className="input-percent-label">%</span>
+    <span className="mr-1 mb-4 align-self-end">%</span>
```

Comment:
[curious] I understand you're keeping this as close to current implementation as possible, but I'm pretty sure this pattern is largely attempting to replicate something that `InputGroup` supports, e.g.:

```jsx
<div>
  <Form.Label htmlFor="example-form-control">An example label</Form.Label>
  <InputGroup>
    <FormControl id="example-form-control" aria-describedby="example-input-group-addon" />
    <InputGroup.Append>
      <InputGroup.Text id="example-input-group-addon">%</InputGroup.Text>
    </InputGroup.Append>
  </InputGroup>
</div>
```

`InputGroup.Text` gives the gray background, but you could use a similar implementation without `InputGroup.Text` which would result in the `%` without a gray background.

<img width="937" alt="image" src="https://user-images.githubusercontent.com/2828721/204891069-27039d87-fb57-46e0-a892-e25337286924.png">

```jsx
<div>
  <Form.Label htmlFor="example-form-control">An example label</Form.Label>
  <InputGroup className="align-items-center">
    <FormControl id="example-form-control" aria-describedby="example-input-group-addon" />
    <InputGroup.Append>%</InputGroup.Append>
  </InputGroup>
</div>
```

<img width="941" alt="image" src="https://user-images.githubusercontent.com/2828721/204891425-f08daff5-1dae-4ef7-a8a5-c6a9c00241d5.png">